internal/wasm/jit/jit_test.go

```diff
@@ -1,147 +1,13 @@
+//go:build !amd64 && !arm64
```

Comment:
If this file builds only when !amd64 && !arm64, then maybe the 12th line isn't necessary anymore?
```go
if runtime.GOARCH != "amd64" && runtime.GOARCH != "arm64" {
```

Reply:
good point though if on a wrong arch I think that TestMain is better (as it covers the whole package)? maybe revert the compile flags and leave it instead? Either way though, remove one or the other.